alpa/pipeline_parallel/pipeshard_executable.py

```diff
@@ -376,6 +376,11 @@ def dump_debug_info(self, folder: str):
             for task in self.resharding_tasks:
                 f.write(str(task) + "\n\n")
 
+        with open(f"{prefix}_input_placement_specs.txt", "w") as f:
+            f.write(str(self.get_input_placement_specs()) + "\n\n")
+        with open(f"{prefix}_output_placement_specs.txt", "w") as f:
+            f.write(str(self.get_output_placement_specs()) + "\n\n")
```

Comment:
Why are there two "\n"?

Reply:
i got it from copying the resharding task on 377 above but it's not needed for single file writing. let me update it.